client/src/App.scss

```diff
@@ -1,5 +1,24 @@
+$font-family-base: "Inter", system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 @import '~bootstrap/scss/bootstrap.scss';
 
 body {
   padding-top: 5rem;
 }
+
+.nav-link {
+  color: white;
+  padding-left: 2rem !important;
+  padding-right: 2rem !important;
+
+  &.active {
+    font-weight: bold;
+    text-decoration: underline;
+  }
+}
+
+.display-1 {
```

Comment:
Here's an example of overriding the default Bootstrap display-1 heading per the design in Figma. You'll need to do this with other styles used in the site.